src/main/java/org/lk/WebController/WebAuthController.java

```diff
@@ -0,0 +1,52 @@
+package org.lk.WebController;
+
+import lombok.AllArgsConstructor;
+import org.lk.model.dto.AuthorizationDto;
+import org.lk.model.dto.InfoAndAuthDto;
+import org.lk.service.AuthorizationService;
+import org.lk.service.InfoService;
+import org.lk.service.SesionService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@AllArgsConstructor
+public class WebAuthController {
+
+    AuthorizationService authorizationService;
```

Comment:
Use `private final` for such fields. After adding `private final` change @AllArgsContructor annotation to @RequiredArgsConstructor